Author: Alexey Svistunov <svalx@svalx.net>
Date: 2013-03-29 16:51 +0400
Description: add support for Panasonic digital camera RAW images.
Upstream: dead 
===================================================================
--- data/mime-type-list.txt.orign	2013-03-29 16:49:15.953376911 +0400
+++ data/mime-type-list.txt	2013-03-29 16:50:28.995095533 +0400
@@ -7,3 +7,5 @@
 image/x-sony-arw
 image/x-epson-erf
 image/x-minolta-mrw
+image/x-panasonic-raw
+image/x-panasonic-raw2
